tests/test_core.py

```diff
@@ -1636,7 +1636,11 @@ def run(*args, **kwargs):
 def test_empty_output(package):
     package.strict = False
     package.set_output_variables()
-    res = package.run(out_dynamic_vars=[])
+
+    if package.version == 7:
+        res = package.run(out_dynamic_vars=[""])
```

Comment:
Why the empty string vs empty array?

Reply:
There was some regression in f90nml which meant it wouldn't write properly if I passed an empty list...